wapm-toml/src/lib.rs

```diff
@@ -561,14 +591,17 @@ mod command_tests {
 
 #[cfg(test)]
 mod dependency_tests {
-    use crate::data::manifest::{Manifest, MANIFEST_FILE_NAME};
-    use crate::util::create_temp_dir;
-    use std::fs::File;
-    use std::io::Write;
+    use super::*;
+    use std::{fs::File, io::Write};
+    use toml::toml;
 
     #[test]
+    #[cfg_attr(
+        feature = "integration_tests",
+        ignore = "Requires the actual Manifest::find_in_directory() implementation which has been mocked out"
+    )]
     fn add_new_dependency() {
-        let tmp_dir = create_temp_dir().unwrap();
```

Comment:
I believe using `create_temp_dir` was intended as otherwise `wapm-cli` would not compile to WASI (since temp dirs are not implemented there) since we need to do compile the wapm-cli to use it as-is in the wapm.io website shell

<img width="1066" alt="Screen Shot 2022-08-17 at 10 56 52 AM" src="https://user-images.githubusercontent.com/188257/185078202-bd70a6dc-9c34-49e0-83ff-d1854beaae62.png">

Reply:
This is correct.

Reply:
No, this is because `create_temp_dir` was copied from wasmer (`crate::util` does not exist).

Reply:
Yeah, `wapm_cli::util::create_temp_dir()` [just defers](https://github.com/wasmerio/wapm-cli/blob/42cfb48b6607c84cd8b5d38d06773c5a85826969/src/util.rs#L309-L311) to the `tempdir` crate so I inlined the call. 

You normally run `cargo test` on a Windows/Linux/MacOS host anyway, so platform compatibility isn't an issue.